### DISCUSSION TITLE
Draggable & Resizable Dialogs

### DISCUSSION THREAD

**Discussion by Imadozone** [quasarframework/quasar#15316] [Ideas / Proposals] [answered]
Hello guys,

It is just a suggestion to implement this feature as it is really needed by quasar users.

Regards,
Imad

**Comment by pdanpdan** [comment, marked as answer]
https://codepen.io/pdanpdan/pen/NWGBPjo

**Reply by Imadozone** [reply]
First of all, thank you very much!
However, with my great appreciation to what the team is doing for us, if it is an option like :is-draggable ="true" or "false" would be much more concise and easier to use.

Also, what about resizable option?

Regards

**Reply by tristanradams** [reply]
I am also curious is there a simple way to add resizable functionality to the QDialog, I've tried my hand at it a few times and can't seem to nail it down. Ideally, just being able to click the bottom right corner for example and drag it to a new size

**Comment by pdanpdan** [comment]
Both draggable and resizable are too specific cases. It would add a lot of code and they do not fit the general use.
Also it is easy to do it custom, exactly for your case
